ipa/ipa_cloud.py

```diff
@@ -598,6 +598,19 @@ def put_file(self, client, source_file):
         else:
             return file_name
 
+    def _cleanup_instance(self, status):
+        """
+        Cleanup instance based on arguments.
+
+        If tests pass and cleanup flag is none, or
+        cleanup flag is true, terminate instance.
+        """
+        if status == 0 and self.cleanup is None or self.cleanup:
```

Comment:
Could you please group the conditions such that we are not bound to the binding strength of the operator, i.e.

```
if (status == 0 and self.cleanup is None) or (self.cleanup):
```